.github/ISSUE_TEMPLATE/discussion.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "\U0001F914 Language Improvement Discussion"
+about: I want to start a new discussion about improving the language.
+labels: type:question
```

Comment:
This label is typically assigned to issues that should have been posted on the forum instead. The issue tracker should be used for actionable issues. That's bug reports and feature discussions, not general questions.

Reply:
I'll put back discussion for the moment then.